odoo/addons/base/models/ir_qweb_fields.py

```diff
@@ -257,7 +258,7 @@ def value_to_html(self, value, options):
 
         if options.get('time_only'):
             format_func = babel.dates.format_time
-            return pycompat.to_text(format_func(value, format=pattern, locale=locale))
+            return pycompat.to_text(format_func(value, format=pattern, tzinfo=tzinfo, locale=locale))
```

Comment:
Would be nice to add a small explanation about this change in the commit message.
I don't think it's worth splitting but let's at least mention it.

Reply:
Not sure what happened but it seems that this:

> I would still split the new converter in a separate commit though, that comes before the changes in event.

Is no longer the case  ^^

Reply:
Oups...

Reply:
And now we are back to the state where I first posted my comment, with an undocumented change.
I'm a bit confused, I'm probably missing something ^^